simperby/src/lib.rs

```diff
@@ -86,9 +92,81 @@ pub struct ConsensusVoteItem {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 pub struct GenesisInfo {
     pub header: BlockHeader,
+    pub genesis_signature: FinalizationProof,
```

Comment:
1. It is redundant to have "genesis" in the field name; the name of the struct implies it.
2. "signature" could be misleading; it is a list of pubkeys and signatures, rather than a single signature.

How about `finalization_proof`?

Reply:
I saw you've fixed it.